psh/tools/netset.py

```diff
@@ -0,0 +1,166 @@
+# Phoenix-RTOS
+#
+# phoenix-rtos-tests
+#
+# Useful functions connected with ethernet setup
+#
+# Copyright 2023, 2024 Phoenix Systems
+# Author: Damian Modzelewski
+#
+# This file is part of Phoenix-RTOS.
+#
+# %LICENSE%
+#
+
+from typing import Optional
+import psh.tools.psh as psh
+import subprocess
+import re
+import time
+from typing import Tuple
+
+
+def ip_table() -> Tuple[str, ...]:
+    return (
+        "192.168.72.20",
+        "192.168.72.21",
+        "192.168.72.120",
+        "192.168.72.121",
+    )
+
+
+def host_interface():
+    out = subprocess.run(
+        "ifconfig",
+        encoding="ascii",
+        stdout=subprocess.PIPE,
+        stderr=subprocess.STDOUT,
+        check=True,
+    )
+
+    eth_device = re.findall("eth[0-9]:.+\n.+[0-9]+", out.stdout)
+    inet = re.findall("inet [0-9.]+", str(eth_device))
+    netmask = re.findall("netmask [0-9.]+", str(eth_device))
+
+    host_setup = inet + netmask
+    return host_setup
```

Comment:
maybe, but do whatever you think is more readable
```suggestion
    return  inet + netmask
```